app/Controller/CoPetitionsController.php

```diff
@@ -872,7 +886,10 @@ protected function dispatch($step, $id=null) {
         'action'     => $this->nextSteps[$step],
         $ptid
       );
-      
+      // RCIAM-57
+      if(isset($target_service_url) && $target_service_url != ""){
```

Comment:
Check if valid url

Reply:
This should only happen at the entry point and not on every occasion.

Reply:
Done